tests/integration/workflows/test_edit_article.py

```diff
@@ -383,3 +385,94 @@ def test_edit_article_callback_does_not_have_redirect_url_if_no_ticket_id(
     assert response.status_code == 200
     data = json.loads(response.data)
     assert 'redirect_url' not in data
+
+
+def test_edit_article_workflow_sending_to_hep(workflow_app, mocked_external_services):
+    app_client = workflow_app.test_client()
```

Comment:
do we an test case if hep replies with ``500``?

Reply:
There are no tests for 500, only for 401, but all error codes are handled the same way.